src/web-worker/web-worker.js

```diff
@@ -35,19 +35,26 @@ self.addEventListener('message', function(event) {
 
   // TODO(choumx): Add error reporting.
   switch (method) {
-    case 'bind.initialize':
-      evaluator_ = new BindEvaluator();
-      returnValue = evaluator_.setBindings.apply(evaluator_, args);
+    case 'bind.addBindings':
+      evaluator_ = evaluator_ || new BindEvaluator();
+      returnValue = evaluator_.addBindings.apply(evaluator_, args);
+      break;
+    case 'bind.removeBindings':
+      if (evaluator_) {
+        returnValue =
+          evaluator_
+          .removeBindingsWithExpressionStrings.apply(evaluator_, args);
```

Comment:
Move `evaluator_.removeBindingsWithExpressionStrings` to a local var to fix this formatting.